tests/elis_api_client/test_annotations.py

```diff
@@ -239,6 +239,56 @@ def is_imported(annotation):
 
         sleep_mock.assert_called_once_with(2)
 
+    async def test_poll_annotation_until_imported(self, elis_client, dummy_annotation):
+        client, http_client = elis_client
+        in_progress_annotation = {**dummy_annotation, "status": "importing"}
+        # First, return annotation in importing, than to_review state
+        http_client.fetch_one.side_effect = [in_progress_annotation, dummy_annotation]
+        # Return sideloaded content
+        http_client.request_json.return_value = {"content": []}
+
+        with patch("asyncio.sleep") as sleep_mock:
+            annotation = await client.poll_annotation_until_imported(
+                dummy_annotation["id"], sleep_s=2, sideloads=["content"]
+            )
+
+        assert annotation == Annotation(**dummy_annotation)
```

Comment:
Shouldn't this be

```python
assert annotation == Annotation(**{**dummy_annotation, "content": []})
```

as in the test of the sync client?

Reply:
Ha, you've uncovered a side-effect in the tests that caused you didn't have to add `content` to the expected annotation, I've fixed it everywhere :wink: